internal/output/plain.go

```diff
@@ -324,3 +330,30 @@ func trimValue(value string, size int) string {
 	}
 	return value
 }
+
+func shiftColsVal(opts []string) int {
+	for _, opt := range opts {
+		a := strings.TrimPrefix(opt, string(ShiftColsPrefix))
+		if len(a) < len(opt) {
+			n, err := strconv.Atoi(a)
+			if err != nil {
+				logging.Errorf("Cannot get shiftCols value: %v", err)
+				break
+			}
+
+			return n
+		}
+	}
+
+	return 0
+}
+
+func columns(offset int, value string) []string {
+	if offset < 0 {
+		offset *= -1
+		logging.Errorf("Negative shiftCols values are not handled, using postive value %d", offset)
```

Comment:
If we encounter this case we should just use 0, if the intend is to fail as gracefully as possible.